cpp/arcticdb/storage/test/test_storage_exceptions.cpp

```diff
@@ -45,25 +45,30 @@ class LMDBStorageFactory : public StorageFactory {
 private:
     uint64_t map_size;
     bool use_mock;
+    fs::path db_path;
+    std::string lib_name;
 public:
-    explicit LMDBStorageFactory(bool use_mock = false) : map_size(128ULL * (1ULL << 20) /* 128MB */), use_mock(use_mock) { }
+    explicit LMDBStorageFactory(uint64_t map_size, bool use_mock = false) : map_size(map_size), use_mock(use_mock), db_path(TEST_DATABASES_PATH / "test_lmdb"), lib_name("test_lib") { }
```

Comment:
I think the defaults here "test_lib" and TEST_DATABASES_PATH / "test_lmdb" are dangerous, remove them?